.vscode/settings.json

```diff
@@ -76,7 +76,8 @@
     "editor.tabSize": 2,
     "editor.formatOnType": true,
     "editor.formatOnPaste": true,
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "biomejs.biome"
```

Comment:
I have also tried using Biome as the formatter, but Biome does not yet support formatting `.md` and `.mdx`files, so we may still need to preserve prettier to format unsupported files.

Reply:
see https://biomejs.dev/internals/language-support/

Reply:
We can configure the formatter of md(x) files as prettier